gwy/templates/main.go.tmpl

```diff
@@ -254,6 +258,18 @@ func SetupMux(ctx context.Context, cfg proxyConfig) *http.ServeMux {
 	logrus.Infof("Proxying requests to gRPC service at '%s'", cfg.backend)
 
 	opts := []grpc.DialOption{grpc.WithInsecure()}
+	
+	if cfg.maxCallRecvMsgSize > 0 || cfg.maxCallSendMsgSize > 0 {
+		callOptions := []grpc.CallOption{}
+		if cfg.maxCallRecvMsgSize > 0 {
```

Comment:
Why we are checking this twice?
Could you remove `if cfg.maxCallRecvMsgSize > 0 || cfg.maxCallSendMsgSize > 0 {` check and ...